packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1191,6 +1191,8 @@ export const prepareVaultManagerKit = (
               debtBrand,
               collateralBrand,
             );
+          state.lockedQuote = undefined;
```

Comment:
consider a comment explaining why this is necessary, and why here. 

EDIT: after reading the test I see it documents this case pretty well